.github/CODEOWNERS

```diff
@@ -2,11 +2,20 @@
 # last matching pattern is used, see
 # https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#codeowners-syntax
 
-* @oss-review-toolkit/kotlin-devs
-*.md @oss-review-toolkit/core-devs
-/**/*Sw360* @oss-review-toolkit/kotlin-devs @bs-ondem @oheger-bosch
-/advisor/ @oss-review-toolkit/kotlin-devs @MarcelBochtler @oheger-bosch
-/clients/fossid-webapp/ @oss-review-toolkit/kotlin-devs @nnobelis
+# Members of @oss-review-toolkit/kotlin-devs.
```

Comment:
I'd prefer to not quote any https://github.com/orgs/oss-review-toolkit/teams names here as I wanted to go through these with @oss-review-toolkit/tsc soon to clean up some redundancy and confusion there.

But we should probably document the general idea behind this "catch all" entry, like by saying "General owners of the Kotlin code base."

Reply:
How about first discussing how we want to organize this and then updating the descriptions in the file afterward? AFAIK we also have no public list of people who are committers (according to the role described in the charter) and if the "owners of legally relevant documents" proposed below are not equal to the TSC members this also needs to be documented somewhere.

Reply:
Of course I'm fine with discussing the GitHub team structure first. But even after that I'd not quote the (new) names here, but because it creates maintenance effort if teams change again.